test/e2e/playwright/benchmark/page-load-benchmark.spec.ts

```diff
@@ -0,0 +1,46 @@
+import path from 'path';
+import { promises as fs } from 'fs';
+import { test as pwTest, expect } from '@playwright/test';
+import { PageLoadBenchmark } from '../../page-objects/benchmark/page-load-benchmark';
+
+pwTest.describe('Page Load Benchmark', () => {
+  let benchmark: PageLoadBenchmark;
+  const outputPath = path.join(
+    process.cwd(),
+    '/test-artifacts/benchmarks/benchmark-results.json',
+  );
+
+  pwTest.beforeAll(async () => {
+    await fs.mkdir(`${process.cwd()}/test-artifacts/benchmarks`, {
+      recursive: true,
+    });
+    const extensionPath = path.join(process.cwd(), 'dist', 'chrome');
+    benchmark = new PageLoadBenchmark(extensionPath);
+    await benchmark.setup();
+  });
+
+  pwTest.afterAll(async () => {
+    await benchmark.cleanup();
+  });
+
+  pwTest('Run page load benchmark', async () => {
+    const testUrls = ['https://metamask.github.io/test-dapp/'];
```

Comment:
just wondering if this could be changed to using a localhost dapp as we use in our tests. This way, we can avoid any external inconsistency coming from this page (ie github pages are down/slow etc..). It could be a follow-up PR if this is seen as a problem over time, or if you consider it's worth it  :pray: